### ISSUE TITLE
Please Update For SC2.

### ISSUE THREAD

**Issue by anusman** [marcelo-mason/Dynmap-SimpleClans#9]
Title says it all. You said you'd work it, but that was months ago.

**Comment by maxammann** [comment]
Yeah sry there is another dev who wanted to work on it, I'm not sure wheater he still wants to, but I'll last him, since I don't use dynmap and I'm not experienced with it

**Comment by anusman** [comment]
That's fine. Just wanted an update. :)
